iphone/Classes/KrollBridge.m

```diff
@@ -770,7 +770,7 @@ -(id)loadCommonJSModule:(NSString*)code withSourceURL:(NSURL *)sourceURL
 	 *      "/Resources/views/login/window.js"  __dirname = "views/login"
 	 */
 
-	NSString *filename = [sourceURL lastPathComponent];
+	NSString *filename = [sourceURL lastPathComponent]; // FIXME This should be the absolute path to the file!
```

Comment:
@sgtcoolguy so now with this fixed here: https://github.com/appcelerator/titanium_mobile/pull/8098
do we still need to address this? @hansemannn fyi.